litellm/llms/ollama_chat.py

```diff
@@ -120,6 +120,28 @@ def get_config(cls):
             and v is not None
         }
 
+# Usage metrics are only populated when the ollama response indicates `"done": true`
+# https://github.com/jmorganca/ollama/blob/main/docs/api.md#generate-a-chat-completion
+class OllamaChatUsage:
+    def __init__(self, response_json):
+        self.response_json = response_json
+
+    def get_usage(self):
+        if self.response_json["done"] == False:
+            return litellm.Usage(
+                prompt_tokens=None,
+                completion_tokens=None,
+                total_tokens=None,
+            )
+
+        prompt_tokens = self.response_json["prompt_eval_count"]
```

Comment:
`prompt_eval_count` cannot be found, but there is `eval_count` in the JSON response.
![image](https://github.com/BerriAI/litellm/assets/8375129/81da2539-b9f1-4d4b-8499-dd795c8cb2ed)